### ISSUE TITLE
fetal anemia

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7009]
**Preferred term label:**
fetal anemia

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**
low hemoglobin/hematocrit in a fetus as evidenced by percutaneous umbilical cord sampling and elevated middle cerebral artery dopplers

**Parent term (use hpo.jax.org/app)**
N/A

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
alloimmunization during pregnancy
hereditary anemias ie alpha thalassemia and PIEZO1-mediated disease
Down syndrome
inborn errors of metabolism
viral infections in a fetus
fetomaternal hemorrhage
other single gene disorders leading to non-immune hydrops

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by saintfrison** [comment]
Would it be useful to specify that  Parvovirus B19 is the virus most often responsible for fetal anemia ?

**Comment by sparkste** [comment]
Don't think it hurts to specify that parvo is the most common etiology. I sometimes hesitate listing examples because the differential diagnosis is so broad though. Happy to discuss further!



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by patt1morgan** [comment]
@wtonyparks, @EDempseyUK.   Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by pnrobinson** [comment]
added
